handler/src/main/java/io/netty/handler/ssl/FixedSslContextSupplier.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import io.netty.util.Supplier;
+import io.netty.util.internal.ObjectUtil;
+
+/**
+ * A {@link Supplier} for {@link SslContext}s.
+ */
+final class FixedSslContextSupplier implements Supplier<SslContext> {
```

Comment:
> is there a roadmap for Netty being JDK8+?

This is not planned for 4.x. However we will very likely raise the minimum required JDK version for future versions of Netty (see https://github.com/netty/netty/issues/5430#issuecomment-227178314))